spec/lib/twingly/url_spec.rb

```diff
@@ -102,6 +114,16 @@ def valid_urls
       end
     end
 
+    context "IDNA 2003 vs IDNA 2008" do
```

Comment:
You should add the `it` yourself when reading, it seems to be what RSpec does in their examples, check for instance:

https://www.relishapp.com/rspec/rspec-core/v/3-5/docs/example-groups/basic-structure-describe-it where you have `it "does something"` and https://www.relishapp.com/rspec/rspec-core/v/3-5/docs/example-groups/shared-examples where you can find `it "should return...."`